Assets/Scripts/Wander.cs

```diff
@@ -0,0 +1,45 @@
+﻿using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+using UnityEngine.AI;
+
+public class Wander : MonoBehaviour
+{
+    [SerializeField]
+    private float wanderRadius;
+
+    [Tooltip("How long before a new destination is randomized?")]
+    [SerializeField]
+    private float wanderTimer;
+
+    //private Transform target;
+    private NavMeshAgent agent;
+    private float timer;
+    
+    void OnEnable()
+    {
+        agent = GetComponent<NavMeshAgent>();
+        timer = wanderTimer;
+    }
+    
+    void Update()
+    {
+        timer += Time.deltaTime;
+
+        if (timer >= wanderTimer)
+        {
+            Vector3 newPos = RandomNavSphere(transform.position, wanderRadius, -1);
+            agent.SetDestination(newPos);
+            timer = 0;
+        }
+    }
+
+    public static Vector3 RandomNavSphere(Vector3 origin, float dist, int layermask)
```

Comment:
you should add a summary for this, especially because it's public and presumably called from other classes.

Reply:
And if it's not necessary to call from outside classes, make it private!